source/server/listener_impl.h

```diff
@@ -74,12 +76,130 @@ class ListenSocketFactoryImpl : public Network::ListenSocketFactory,
 // TODO(mattklein123): Consider getting rid of pre-worker start and post-worker start code by
 //                     initializing all listeners after workers are started.
 
+/**
+ * The immutable factory context during the intelligent listener update. The continuous intelligent
```

Comment:
What is an intelligent listener? Wrong comment?

Reply:
The comment is back from future :)

Fixing